src/slskd/Transfers/Uploads/UploadService.cs

```diff
@@ -24,13 +24,15 @@ namespace slskd.Transfers.Uploads
     using System;
     using System.Collections.Concurrent;
     using System.Collections.Generic;
+    using System.Diagnostics.Eventing.Reader;
```

Comment:
```suggestion
    using System.Diagnostics.Eventing.Reader;
```

this shouldn't be necessary.  if you're using VS Code it may have added it automatically

Reply:
Fixed. Thank you.